utils/middlewares/service.ts

```diff
@@ -0,0 +1,45 @@
+import { IRequestMiddleware } from "io-functions-commons/dist/src/utils/request_middleware";
+import { ResponseErrorFromValidationErrors } from "italia-ts-commons/lib/responses";
+
+import { Logo as ApiLogo } from "../../generated/definitions/Logo";
+import { ServicePayload } from "../../generated/definitions/ServicePayload";
+import { SubscriptionKeyTypePayload } from "../../generated/definitions/SubscriptionKeyTypePayload";
+
+/**
+ * A middleware that extracts a Service payload from a request.
+ */
+export const ServicePayloadMiddleware: IRequestMiddleware<
```

Comment:
Can we use RequiredBodyPayloadMiddleware?

Reply:
Fixed in [e040495](https://github.com/pagopa/io-functions-services/pull/68/commits/e040495a4536f99dc6b9b4132552e8959ad1dae0)